ui/admin/app/routes/scopes/scope/session-recordings/session-recording/channels-by-connection.js

```diff
@@ -48,4 +51,20 @@ export default class ScopesScopeSessionRecordingsSessionRecordingChannelsByConne
       storageBucket,
     };
   }
+
+  /**
+   * Reapplies storage policy dates to session recording
+   * @param {SessionRecordingModel}
+   */
+  @action
+  @notifyError(({ message }) => message, { catch: true })
+  @notifySuccess(({ state }) =>
+    state === STATE_SESSION_RECORDING_AVAILABLE
```

Comment:
If the session is in an error or unknown state, should it also say later?

Reply:
thanks for raising this- checking with BE, for now, we show the manage dropdown, not sure if the user can still reapply at that state

Reply:
update: yes, we will show it as later